package.json

```diff
@@ -4,7 +4,10 @@
   "description": "- Nodejs 7.x\r - npm: discord.js",
   "main": "index.js",
   "scripts": {
-    "test": "mocha"
+    "start": "node src/bot.js",
```

Comment:
I want to leave it likes this. test is used by travis directly and start is used by the container, I just added start:docker and test:docker to make running those container easier for our own benefit.

Reply:
Can we not just swap the values in references? I'm just thinking the local version is going to be the one we're typing and I'd rather type start 5000 times than start:docker

Reply:
You can use up arrow to reuse previous commands you entered. Also in ide's(at least Webstorm) you can add a script which you can run with the press of a button. Honestly I would like to look at a solution(and I know it is possible) where it detects changes and then restarts the bot with the new code.

Besides, instead of building a container each time we make changes(even with use of the docker cache now) we should really look into just running the bot locally without docker. The only problem is that one of the dependencies uses python and needs to compile some stuff (and I have no idea how to fix it). To make sure that we run the bot with the intended node version we can use nvm, tho running it with a newer version should not bring any problems (just it will support newer features, which will fail during testing when implemented).

Reply:
As the section currently stands if we change ```test``` --> ```mocha``` and ```test:docker``` --> ```test``` I think this would be good to merge.

Automatic rebuilding would be cool, but running the bot outside of our docker environment would be very bad for consistency. The whole point of docker setup is to hopefully provide identical environments on all systems. You are of course free to do whatever you want locally but I would recommend against it

Reply:
Differences between platforms and versions are very minimal as far as I can tell. We should probably set some requirement conditions for which the bot should work with (like the nodejs version). We can always expand it when we find problems, but we don't really want to trade ease of development for avoidance of possible problems we might encounter. Also testing should catch any problems and else be expanded to catch those problems. I already encountered this when I used finally on a promise.